hive-runtime/NOTICE

```diff
@@ -42,11 +42,11 @@ NOTICE file:
 |     * Redistributions of source code must retain the above copyright
 | notice, this list of conditions and the following disclaimer.
 |     * Redistributions in binary form must reproduce the above
-| copyright notice, this list of conditions and the following disclaimer
-| in the documentation and/or other materials provided with the
+| copyright notice, this list of conditions, and the following disclaimer
+| In the documentation and/or other materials provided with the
```

Comment:
```suggestion
| in the documentation and/or other materials provided with the
```
I think it would be better to leave this part as it is rather than start a new sentence.